subsys/partition_manager/Kconfig

```diff
@@ -102,6 +102,19 @@ module=802154_RPMSG
 source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.build_strategy_domain"
 endif
 
+if BT_RPC && SOC_NRF5340_CPUAPP
+module=RPC_HOST
+source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.build_strategy_domain"
+
+DT_CHOSEN_Z_IPC_SHM := zephyr,ipc_shm
+
+config BT_RPC_HOST_NRF53_SRAM_SIZE
+	# Hidden configuration to expose DT property as kconfig variable
+	hex
```

Comment:
Will this configuration never change? If the user should be able to modify it, it should be exposed here instead of through DT as partition manager "takes ownership" of these configurations in NCS. Using the default from DT makes sense anyway.

Reply:
I modeled here on the RPMSG_NRF53 because I want to achieve a similar behaviour for this sample. I don't think this configuration is going to change

Reply:
If its not supposed to be modified by users then this is fine.